### ISSUE TITLE
No Update Notification After Upgrading to 5.10.x

### ISSUE THREAD

**Issue by link470** [invoiceninja/ui#1976]
In InvoiceNinja 5.8.x and below, I had an update notification in the form of a blue triangle in the bottom left of the screen where the Info button is.  In the React UI, this is now gone, and there's no indication of a later version being available.

In the iOS app, the update notification is still there and, when tapped, correctly displays the current version I'm on (5.10.16) and the newest version available (5.10.20), as shown below:

![607778deea9a2ceb82f014c15994d7f08111e2a8](https://github.com/user-attachments/assets/f724c666-2b28-4698-807e-020a15eb7a8b)

Further, the React UI now just shows _v Latest Build_, but it's not actually the latest version since I'm 4 versions behind:

![a18cd0124f664f43478c905c532af3a9def152d8](https://github.com/user-attachments/assets/047df821-1b82-4c95-8073-d723272c8378)

For more information, see the ongoing thread here on the InvoiceNinja forums: https://forum.invoiceninja.com/t/new-updates-is-not-displayed-in-the-new-ui/16372

**Comment by turbo124** [comment]
The react UI is built from the develop channel, you'll want to use the Health check to see the API version as that will contain the reference you are more familiar with v5.10.xxxx

We'll bring in the notification triangle to the UI.

**Comment by link470** [comment]
> you'll want to use the Health check to see the API version as that will contain the reference you are more familiar with

Gotcha, I understand that.  But something still doesn't seem right if the React UI Info window is displaying "v Latest Build", because the version of the React UI that came with 5.10.16 (that I'm currently running) is _not_ the latest build, is it?

It went from displaying the actual version:

![352881199-604974ea-e86d-4cd4-9f3e-b9d72c884dd0](https://github.com/user-attachments/assets/d54cc2fe-9990-479a-a7df-3cef3fb53716)

To displaying "Latest Build":

![357124911-047df821-1b82-4c95-8073-d723272c8378](https://github.com/user-attachments/assets/6f74b602-39fc-4906-80ad-e1e0093ac768)

If 5.10.16's version of the React UI isn't in fact the latest React UI build, it feels odd that it displays "Latest Build", and that seems like a bug.  But, if the version of the React UI hasn't changed in the latest version of InvoiceNinja (5.10.20), then this is correct.

**Comment by turbo124** [comment]
The react tag has never been correct. Previously the react release tag referred to the last "tagged" release of the UI, but this was incorrect.

Our build pipeline works like this:

When we tag a release of the API, ie v5.10.20 the build pipeline packages up the latest version of the develop branch of the React UI. That is why we changed the description to v latest build and inserted the build date.

**Comment by gaetan-hexadog** [comment]
> The react tag has never been correct. Previously the react release tag referred to the last "tagged" release of the UI, but this was incorrect.
> 
> Our build pipeline works like this:
> 
> When we tag a release of the API, ie v5.10.20 the build pipeline packages up the latest version of the develop branch of the React UI. That is why we changed the description to v latest build and inserted the build date.

Thanks for the clarification about the pipeline. But regarding this behavior displaying `v Latest Build` doesn't brings no value due to the fact it's always `v Latest Build` no matter what the real version number is.

If the pipeline doesn't allow to retrieve the real version number to being able to display it. Maybe it would be better to not display it at all and just keep the release date.

**Comment by turbo124** [comment]
@gaetan-hexadog the react version never has a version in this context. Its version is tightly coupled to the API release, so that should be used as the reference for the React Version.

**Comment by gaetan-hexadog** [comment]
@turbo124 that's why I think it might be preferable not to display it at all, and to only keep the release date.
Because there is no value to always display `v Latest Build` (no matter what version we are trying to display) if it's not changing and if we can't compare it with any available version, ...

**Comment by link470** [comment]
> @turbo124 Because there is no value to always display `v Latest Build` (no matter what version we are trying to display) if it's not changing and if we can't compare it with any available version, ...

I agree.  Otherwise, it's just confusing checking the status and seeing "v Latest Build", thinking "oh, I'm up to date", when really, I'm not up to date at all, and there's no indication here that this v Latest Build text is related to React.  At first glance, seeing this makes me think I'm up to date, when in fact I'm 8 versions behind:

![357269414-6f74b602-39fc-4906-80ad-e1e0093ac768](https://github.com/user-attachments/assets/a3007987-ed69-433c-9250-46e665682cdb)

**Comment by turbo124** [comment]
That is why we also append the date of the build itself to this particular line.

**Comment by link470** [comment]
This was closed, but I have a feeling this only changed the version text in the About window, _not_ the original issue of no blue update notification triangle appearing in the lower left to alert when a new version is available.  I'm on 5.10.26 and the version looks correct now, but I still don't see a notification icon telling me 5.10.27 is available, so I have a feeling the original issue here might have been missed.

This was originally discussed on the forums here: https://forum.invoiceninja.com/t/new-updates-is-not-displayed-in-the-new-ui/16372

**Comment by turbo124** [comment]
@link470 can you add a new issue for that, i'll tag it into the backlog

**Comment by link470** [comment]
> @link470 can you add a new issue for that, i'll tag it into the backlog

This *is* the issue.  If you read the first post in this, you'll see it describes the exact same thing.  But I'll make another issue with the same text if you wish.

**Comment by link470** [comment]
> Actually, I see I can edit the title. I'll modify this one so it's just talking about the version being displayed in the About window (the issue that was fixed).

On second thought, nevermind.  I'm sticking with my original thought of this being the issue that needs to be fixed still.  This should be reopened, and [THIS issue](https://github.com/invoiceninja/ui/issues/1934) (the one I believe was actually fixed here) should be closed as fixed.  The original issue in the first post here is still a problem, and this should be reopened.

**Comment by link470** [comment]
Can this be re-opened per my message above?

**Comment by link470** [comment]
@turbo124 Just a reminder that this one should be re-opened as the original issue was never fixed.  See my post 2 above this one.  Thanks!

**Comment by turbo124** [comment]
@Civolilah this item is to presnet an upgrade icon notification in place of the (i) button

The logic here is if

```
https://pdf.invoicing.co/api/version != current version - show the upgrade icon
```

**Comment by sjmdia** [comment]
i also intressted to get the update Notifcation Icon back in React app.... 

Forum Post https://forum.invoiceninja.com/t/new-updates-is-not-displayed-in-the-new-ui/16372 by AngryWarrior

the update Nofication Icon ist vissible in FLutter ap but not in React App

**Comment by hillelcoren** [comment]
There are recent reports of this issue in the support forum

https://forum.invoiceninja.com/t/new-updates-is-not-displayed-in-the-new-ui/16372/60?u=hillel

**Comment by turbo124** [comment]
Its been merged here: #2226 just awaiting release.
